### ISSUE TITLE
Don't enforce Javascript style guide

### ISSUE THREAD

**Issue by silvestreh** [vuejs-templates/webpack#67]
I found that it's really frustrating being forced to use a JS style guide. I believe linting errors and warnings should only pop up on real code issues and not for using a different number of spaces for indentation.

I like using semi colons to end statements, this might be a very uncommon example, but the JS engine will interpret the following …

```javascript
return
{
    prop: 'val'
}
```

as

```javascript
return;
{
    prop: 'val'
};
```

By automatically adding its own semi colons and it would actually return nothing instead of the object I was expecting. Having semi colons helps avoiding these kind of issues (although the example I'm using is anything but common)

TL:DR; Don't stop me from using the style that I'm most comfortable with. Ship the template with sane ESLint rules that won't stop me from actually writing code.

**Comment by chrisvfritz** [comment]
@silvestreh I think what's "sane" is very subjective in this case. We're shipping with rules from [a community-developed standard](https://github.com/feross/standard), so it's not arbitrary. The good news is that you can [use your own fork](https://github.com/vuejs-templates/webpack#fork-it-and-make-your-own) with vue-cli, with eslint rules that fit better with the coding style you'd like to enforce. :smiley: Or you can turn off eslint completely.

In the future, we're planning on making linting and some other features optional when generating a new project.

**Comment by silvestreh** [comment]
Yes, it's kind of subjective. That's why I was hoping to have rules that don't enforce a style, but rather print errors and warnings for _actual errors and warnings_. Is having a different setting for indentation an error? I don't think so.

I believe that, at least for now, the fork option is the best one. Thanks!
